util/clientmetric/clientmetric.go

```diff
@@ -223,6 +224,33 @@ func NewGaugeFunc(name string, f func() int64) *Metric {
 	return m
 }
 
+type AggregateCounter struct {
+	mu       sync.RWMutex
+	counters []*expvar.Int
+}
+
+func (c *AggregateCounter) Value() int64 {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	var sum int64
+	for _, c := range c.counters {
+		sum += c.Value()
+	}
```

Comment:
Oops, I used `c` twice here ;)

```suggestion
	for _, cnt := range c.counters {
		sum += cnt.Value()
	}
```